fnet/fnet_model.py

```diff
@@ -67,10 +69,10 @@ def save_state(self, path_save):
         torch.save(self.get_state(), path_save)
         self.to_gpu(curr_gpu_ids)
 
-    def load_state(self, path_load, gpu_ids=-1):
+    def load_state(self, path_load, gpu_ids=-1, model_kwargs={}):
         state_dict = torch.load(path_load)
         self.nn_module = state_dict['nn_module']
-        self._init_model()
+        self._init_model(model_kwargs)
```

Comment:
would all the model_kwargs be better put with the state... why should i have to remember how i initialized the model when i'm reloading it?

Reply:
Agree, model/neural net hyperparameters should definitely go into the save state.